bool gait_toggle
bool hop_toggle
bool joystick_toggle
float32 x               # LEFT-RIGHT movement
float32 y               # FORWARD-BACKWARD movement
float32 roll
float32 pitch
float32 yaw
float32 height_movement
